tools/test-harnesses/src/testUtils/renderAsyncTest.tsx

```diff
@@ -0,0 +1,55 @@
+import React, { useState } from 'react';
+
+interface RenderAsyncTestReturnType {
+  openButton: HTMLButtonElement;
+  asyncTestComponentId: string;
+  [key: string]: any;
+}
+
+export const renderAsyncTest = (
```

Comment:
What does this do? Why do we need this?

Reply:
This my example of a modal-like component. I did this to test that you can find an element inside another element that opens after a delay.